Xamarin.Forms.Core/VisualStateManager.cs

```diff
@@ -17,6 +17,19 @@ public class CommonStates
 			public const string Selected = "Selected";
 		}
 
+		public class TouchStates
+		{
+			public const string Pressed = "Pressed";
+			public const string Released = "Released";
+			public const string Move = "Move";
```

Comment:
I wonder if this should be Moved or Dragged?

Reply:
changed the name to Move.